docs/integrations/azure/locations.md

```diff
@@ -0,0 +1,30 @@
+---
+id: locations
+title: Azure DevOps Locations
+sidebar_label: Locations
+description: Documentation on Azure DevOps location integration
+---
+
+The Azure integration supports loading catalog entities from Azure DevOps.
+Components can be added to
+[static catalog configuration](../../features/software-catalog/configuration.md),
+or registered with the
+[catalog-import](https://github.com/backstage/backstage/tree/master/plugins/catalog-import)
+plugin.
+
+```yaml
+integrations:
+  azure:
+    - host: dev.azure.com
+      token:
+        $env: AZURE_TOKEN
```

Comment:
```suggestion
      token: ${AZURE_TOKEN}
```

i think the time has come to start recommending this form. The separate-object dollar form keeps tripping people up.

The same goes for all examples below as well